sticker-utils/testdata/sticker.conf

```diff
@@ -31,7 +31,6 @@
   inputs_op = "prediction/model/inputs"
   tags_op = "prediction/model/tags"
   seq_lens_op = "prediction/model/seq_lens"
```

Comment:
Yeah, they are always wrong, and it's really annoying. I made these configurable for when somebody wants to define their own graph with custom op names. In hindsight, it might be a bad choice. Maybe I should switch back to hardcoding the op names in the tensorflow module. If someone creates a custom graph, it will still work if they use the same op names. They'd probably modify the existing Python scripts anyway.

Reply:
It's not really important since there's the `.conf` with correct paths under `example/`.

Although if the paths of the ops are changed, chances are that the Rust code also changed through adding, e.g., an additional op. I don't think anyone modifying the graph/model would be done by simply editing the `.conf` anyways, so it should be fine to just put them in the `tensorflow::model` module or wherever the ops reside.